revscoring/scoring/models/tests/test_svc.py

```diff
@@ -0,0 +1,66 @@
+from ..model import Model
+from ..svc import RBFSVC, SVC, LinearSVC
+from .util import FEATURES, format_info, pickle_and_unpickle, train_test
+
+
+def test_svc():
+    model = SVC(FEATURES)
+    format_info(model)
+    train_test(model)
+    pickle_and_unpickle(model)
+    format_info(model)
+
+    model = SVC(FEATURES, scale=True, center=True)
+    format_info(model)
+    train_test(model)
+    pickle_and_unpickle(model)
+    format_info(model)
+
```

Comment:
There are assertions inside the functions.  Were you looking for something else?